packages/headless/src/api/nextFetchFromWP.ts

```diff
@@ -40,23 +40,28 @@ export default async function nextFetchFromWP({
     ? await resolveTemplate(pageInfo as UriInfo, templates)
     : undefined;
 
+  const isLatestPostsFrontPage =
```

Comment:
`isLatestPostsFrontPage` could be part of `UriInfo` if we think it has use outside of this file.

It would enable `pageInfo?.isLatestPostsFrontPage` the same way we can currently do `pageInfo?.isSingular`.